<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creerfacturewindow</class>
 <widget class="QMainWindow" name="creerfacturewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="bouton_enregistrer">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 85, 255);
font-weight:bold;
color:white;</string>
     </property>
     <property name="text">
      <string>Enregistrer</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,127);
color:white;
font-weight:bold;
border-style:solid;
border-color:yellow;</string>
     </property>
     <property name="text">
      <string>      Nouvelle facture</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,127);
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Quantit√©</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_quantite">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,127);
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Num. Commande</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_num_cmd">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,127);
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Num. Prod. Commande</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_num_prod_cmd">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,127);
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Total</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_total">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0,0,127);
font-weight:bold;</string>
     </property>
     <property name="text">
      <string>Prix</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_prix">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>372</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
